model/registry/test.yaml

```diff
@@ -0,0 +1,85 @@
+groups:
+  - id: registry.test.suite
```

Comment:
```suggestion
  - id: registry.test
```

I think this should fix the title in the file

Reply:
Yup, it did, thanks!